### ISSUE TITLE
New release to maven central

### ISSUE THREAD

**Issue by thorstenwagner** [rwl/ParallelColt#11]
Could you update the release on maven central that the latest changes are included?

If not, I can do it but with another group ID: de.biomedical-imaging.parallelcolt

**Comment by rwl** [comment]
Currently experiencing issues performing a new release using Sonatype:

https://issues.sonatype.org/browse/OSSRH-3577

In the meantime, you may wish to release your changes under a different group ID.  Alternatively, you could request permission to upload to Sonatype using your own account.  I'd be happy to endorse the request.

**Comment by thorstenwagner** [comment]
I'm not an expert with sonatype but when I compare your pom.xml with some of mine which are also synced to sonatype, then I'm missing the following lines:

```xml
<build>
		....
		<plugins>
			<plugin>
				<groupId>org.sonatype.plugins</groupId>
				<artifactId>nexus-staging-maven-plugin</artifactId>
				<version>1.6.3</version>
				<extensions>true</extensions>
				<configuration>
					<serverId>ossrh</serverId>
					<nexusUrl>https://oss.sonatype.org/</nexusUrl>
					<autoReleaseAfterClose>true</autoReleaseAfterClose>
				</configuration>
			</plugin>
		</plugins>
	</build>
```

And your release plugin seems to be outdated:

```xml
<groupId>org.apache.maven.plugins</groupId>
					<artifactId>maven-release-plugin</artifactId>
					<version>2.5</version>
```

I'll wait until you can update your release at sonatype...

**Comment by rwl** [comment]
You are right.  The authorization issue was due to a problem with my `settings.xml` file.  The `pom.xml` is no longer compatible with nexus and needs to be updated according to this guide:

http://central.sonatype.org/pages/apache-maven.html
